schema/2.0.0/corpora_schema.md

```diff
@@ -0,0 +1,257 @@
+
+*   **No PII**. Curators agree to this requirement as part of the data submission policy.
+    However, it is not strictly enforced in our validation tooling because it is difficult for software to predict what is and is not PII.
+    It is up to the submitter to ensure that no metadata can be personally identifiable: no names, dates of birth, specific locations, etc.
+    See this [list](https://docs.google.com/document/d/1sboOmbafvMh3VYjK1-3MAUt0I13UUJfkQseq8ANLPl8/edit) for guidance.
+
+#### *Note on types*
+The types below are python3 types. Note that a python3 `str` is a sequence of Unicode code points, which is stored null-terminated and UTF-8-encoded by anndata.
+
+## `X` (Matrix Layers)
+
+cellxgene does not impose any additional constraints on the `X` data matrix.
+
+In any layer, if a matrix has 50% or more  values that are zeros, it is STRONGLY RECOMMENDED that the matrix be encoded as a [`scipy.sparse.csr_matrix`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.csr_matrix.html).
+
+cellxgene's matrix layer requirements are tailored to optimize data reuse. Because each assay has different characteristics, the requirements differ by assay type. In general,
+cellxgene requires submission of "raw" data suitable for computational reuse when a standard raw matrix format exists for an assay and strongly recommends that a "final" matrix suitable for
+visualization in the explorer be included. So that cellxgene's data can be provided in download formats suitable for both R and Python, the schema imposes the following requirements:
+
+*   All matrix layers MUST have the same shape, and have the same cell labels and gene labels.
+*   Because it is impractical to retain all barcodes in raw and final matrices, any cell filtering MUST be applied to both.
+    By contrast, those wishing to reuse datasets require access to raw gene expression values, so genes MUST NOT be filtered from either dataset.
+    Summarizing, any cell barcodes that are removed from the data MUST be filtered from both raw and final matrices and genes MUST NOT be filtered from the raw matrix.
+*   Any genes that publishers wish to filter from the final matrix MAY have their values replaced by a language appropriate "null" value (e.g. [`np.nan`](https://numpy.org/doc/stable/reference/constants.html#numpy.nan) for python), which will mask them from exploration.
+*   Additional layers provided at author discretion MAY be stored using author-selected keys, but MUST have the same cells and genes as other layers.
+
+The following table describes the matrix data and layers requirements that are **assay-specific**. If an entry in the table is empty, the cellxgene schema does not have any other requirements on data in those layers beyond the ones listed above.
+
+| Assay | "raw" required? | "raw" requirements | "final" required? | "final" requirements | Other layers |
```

Comment:
I think it was specifically added to ATAC/methylation-seq because several times we have gotten the question of whether we support non-gene data.

If it makes it confusing we could add a sentence for the sc-RNA seq rows in the ' column "raw" required? ', indicating that values must correspond to ENSEBML genes. 

@brianraymor I think this will be your decision.

Reply:
> I think it was specifically added to ATAC/methylation-seq because several times we have gotten the question of whether we support non-gene data.

2.0.0 only supports "gene" as a `feature_biotype` so it's clear that there is NOT support for non-gene data.  I will delete as redundant. 

I've debated whether to move the `X` section after the discussions of `obs` and `var` which might clarify that point.

Reply:
Addressed in https://github.com/chanzuckerberg/single-cell-curation/pull/67/commits/5395be955930cae03eadd7a66058a38196b83e69